apps/system/js/app_chrome.js

```diff
@@ -898,6 +909,7 @@
 
   AppChrome.prototype.checkForPinnedPage = function ac_checkForPinnedPage() {
     window.places.getPlace(this.app.config.url).then((function(place) {
+      this.currentPlace = place;
```

Comment:
Note that this only gets called when you navigate outside the current scope and the new scope isn't part of a pinned site. It isn't a reliable way to track the current place if that's what you're using it for.